### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by leedenison** [hypothesis/product-backlog#1480]
This issue should eventually be broken into separate issues for accessibility milestones we would like to achieve.  The currently unordered list of issues is:
- [VoiceOver does not tell user they should log in](https://github.com/hypothesis/client/issues/3235)
- [Allow keyboard users to move from annotation card to anchor text and vice versa](https://github.com/hypothesis/client/issues/2061)
- [Investigate making use of ARIA annotations](https://github.com/hypothesis/client/issues/1856)
- [Search result screen reader accessibility issues](https://github.com/hypothesis/h/issues/4557)
- [Cannot focus lozenges using keyboard](https://github.com/hypothesis/h/issues/4079)
- [Search result list cannot be navigated with keyboard](https://github.com/hypothesis/h/issues/3853)
- [SPIKE: Browser focus doesn't shift to annotation when using VoiceOver + Safari for Mac on first annotation](https://github.com/hypothesis/client/issues/2041)
- [Set a useful title on embedded media iframes](https://github.com/hypothesis/client/issues/1959)

**Comment by robertknight** [comment]
Closing this as we have other issues elsewhere that track more concrete accessibility work packages (eg. work needed as part of completing a VPAT, or resolving a collection of issues reported by a particular customer).
